app/views/eventRegistrations.html

```diff
@@ -42,20 +42,46 @@ <h2 class="page-title">Registrations</h2>
             <i class="fa fa-chevron-down" ng-show="!showMoreFilters"></i>
             <i class="fa fa-chevron-up" ng-show="showMoreFilters"></i>
           </button>
-          <button class="btn btn-primary" ng-click="refreshRegistrations()"><i class="fa fa-refresh"></i> Refresh</button>
-          <button type="button" class="btn btn-primary dropdown-toggle" dropdown-toggle>
-            Columns
-            <i class="fa fa-caret-down"></i>
+          <button class="btn btn-primary" ng-click="refreshRegistrations()">
+            <i class="fa fa-refresh"></i>
+            Refresh
           </button>
-          <ul class="dropdown-menu dropdown-menu-right reg-views-dropdown">
-            <li ng-repeat="block in blocks" ng-if="block.profileType !== 'NAME' && block.profileType !== 'EMAIL'">
-              <a href="" ng-click="toggleColumn($index)">
-                <i class="fa fa-check-square" ng-show="block.visible"></i>
-                <i class="fa fa-square-o" ng-show="!block.visible"></i>
-                {{block.exportFieldTitle || block.title}}
-              </a>
-            </li>
-          </ul>
+          <div class="btn-group" role="group" dropdown auto-close="outsideClick">
+            <button type="button" class="btn btn-primary dropdown-toggle" dropdown-toggle>
+              Display Columns
+              <i class="fa fa-caret-down"></i>
+            </button>
+            <ul class="dropdown-menu dropdown-menu-center reg-views-dropdown">
+              <li>
+                <a href="" ng-click="builtInColumnsVisible.Email = !builtInColumnsVisible.Email">
+                  <i class="fa fa-check-square" ng-show="builtInColumnsVisible.Email"></i>
+                  <i class="fa fa-square-o" ng-show="!builtInColumnsVisible.Email"></i>
```

Comment:
I know you didn't initially write this code but all 4 of these icons could be cleaned up using `ng-class`. Something like:
```
<i class="fa" ng-class="builtInColumnsVisible.Email ? 'fa-check-square' : 'fa-square-o'"></i>
```
I hope this works with the version of angular that the ERT is running. If not the [evaluated expression version](https://scotch.io/tutorials/the-many-ways-to-use-ngclass#ngclass-using-evaluated-expression) should work.
I found examples here: https://scotch.io/tutorials/the-many-ways-to-use-ngclass